controller/test-files/search/show/in_topology_A-B-C_and_A-D_search_for/A_with_tree-view=false.res.golden.json

```diff
@@ -0,0 +1,88 @@
+{
+  "data": [
+    {
+      "attributes": {
+        "system.created_at": "0001-01-01T00:00:00Z",
+        "system.number": 1,
+        "system.order": 1000,
+        "system.remote_item_id": null,
+        "system.state": "new",
+        "system.title": "A",
+        "system.updated_at": "0001-01-01T00:00:00Z",
+        "version": 0
+      },
+      "id": "00000000-0000-0000-0000-000000000001",
+      "links": {
+        "related": "http:///api/workitems/00000000-0000-0000-0000-000000000001",
+        "self": "http:///api/workitems/00000000-0000-0000-0000-000000000001"
+      },
+      "relationships": {
+        "area": {},
+        "assignees": {},
+        "baseType": {
+          "data": {
+            "id": "00000000-0000-0000-0000-000000000002",
+            "type": "workitemtypes"
+          },
+          "links": {
+            "self": "http:///api/spaces/00000000-0000-0000-0000-000000000003/workitemtypes/00000000-0000-0000-0000-000000000002"
+          }
+        },
+        "children": {
+          "links": {
+            "related": "http:///api/workitems/00000000-0000-0000-0000-000000000001/children"
+          },
+          "meta": {
+            "hasChildren": true
+          }
+        },
+        "comments": {
+          "links": {
+            "related": "http:///api/workitems/00000000-0000-0000-0000-000000000001/comments",
+            "self": "http:///api/workitems/00000000-0000-0000-0000-000000000001/relationships/comments"
+          }
+        },
+        "creator": {
+          "data": {
+            "id": "00000000-0000-0000-0000-000000000004",
+            "links": {
+              "related": "http:///api/users/00000000-0000-0000-0000-000000000004",
+              "self": "http:///api/users/00000000-0000-0000-0000-000000000004"
```

Comment:
this `self` link should be removed. It's a `related` link, as already included above.